libbeat/docs/dashboardsconfig.asciidoc

```diff
@@ -40,25 +40,28 @@ You can specify the following options in the `dashboards` section of the
 
 ===== enabled
 
-If enabled, load the sample Kibana dashboards on startup. If no other options
-are set, the dashboards archive is downloaded from the elastic.co website.
+If enabled, {beatname_uc} loads the sample Kibana dashboards automatically on startup.
+If no other options are set, the dashboard archive is downloaded from the
+elastic.co website.
 
 ===== url
 
-The URL from where to download the dashboards archive. By default this URL has a
-value which is computed based on the Beat name and version. For released
-versions, this URL points to the dashboard archive on the artifacts.elastic.co
+The URL to use for downloading the dashboard archive. By default this URL
+is computed based on the Beat name and version. For released versions,
+this URL points to the dashboard archive on the artifacts.elastic.co
 website.
 
 ===== directory
 
-The directory from where to read the dashboards. It is used instead of the URL
-when it has a value.
+The directory that contains the dashboards to load. If this option is set,
+{beatname_uc} looks for dashboards in the specified directory instead of
+downloading the archive from a URL. 
 
 ===== file
 
-The file archive (zip file) from where to read the dashboards. It is used
-instead of the URL when it has a value.
+The file archive (zip file) that contains the dashboards to load. If this option
```

Comment:
Is this simply the filename or a path? (My apologies for not testing, but thought it would be faster just to ask.)